stdlib/public/Concurrency/TaskPrivate.h

```diff
@@ -113,6 +115,61 @@ _swift_task_getDispatchQueueSerialExecutorWitnessTable() {
 }
 #endif
 
+/*************** Methods for Status records manipulation ******************/
+
+/// Remove the top status record from input task. This may be called
+/// asynchronously from the current task.  After this call returns, the record's
+/// memory can be freely modified or deallocated.
+///
+/// This must be called synchronously with the task.  The record must
```

Comment:
Whoops - I seem to have contradicted myself on whether this is asynchronous or synchronous from current task. Will fix.